### ISSUE TITLE
Warehouse wise stock balance not showing stock available in other company warehouses

### ISSUE THREAD

**Issue by maveezmh** [frappe/erpnext#35621]
### Information about bug

Warehouse wise stock balance report is supposed to show stock in all warehouses irrespective of companies 

but now its showing only for one company

if you see the screenshot of item master you can see the available stock in different companies. 
![image](https://github.com/frappe/erpnext/assets/112148467/e061a0cb-7ee4-4ebf-bcc5-bff4e455c944)

but in Warehouse report its showing stock for only one company. it was working fine earlier but now ita not. 
![image](https://github.com/frappe/erpnext/assets/112148467/090fc4cc-53f1-44c4-b39d-92fcc4ee002b)

Basically somehow the report is fetching and filtering company from session defaults and showing stock only for selected company

![image](https://github.com/frappe/erpnext/assets/112148467/ca931cff-f134-433c-a30a-3e155b32df26)

### Module

stock

### Version

V13 and V14

### Installation method

FrappeCloud

**Comment by s-aga-r** [comment]
Broken from: https://github.com/frappe/erpnext/pull/35348

**Comment by maveezmh** [comment]
Any Update

**Comment by maveezmh** [comment]
@rohitwaghchaure any update?

**Comment by maveezmh** [comment]
@rohitwaghchaure @s-aga-r any update on this report??
